docs/contributing/contributing.md

```diff
@@ -3,34 +3,76 @@ id: contributing
+1. Setup a development environment by following the brief [Dev Setup Guide](dev-setup) or the more detailed [Tutorial](../tutorials/tut-setup-dev).
+1. Review the [Contribution Checklist](../contributing/checklist).
+1. Follow the [Content Pack format](packs-format) to build your contribution. [demisto-sdk init](https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/init/README.md) will help you create it.
+1. Depending on the content entities you need to build, navigate to the specific section of this website for details. If you are creating Integrations and/or Automations, make sure that you:
+    * Use the proper  [Directory Structure](../integrations/package-dir). [demisto-sdk init](https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/init/README.md) will help you create it. If working on existing code, beyond trivial changes, we require converting to this structure as it allows running linting and unit tests and provides a clearer review process.
+    * Understand the [YAML file](../integrations/yaml-file) structure and the [Parameter Types](../integrations/parameter-types).
+    * Make sure your integration follows our [Logo Guidelines](../integrations/integration-logo).
+    * Read and follow [Python code conventions](../integrations/code-conventions) (recommended) or [Powershell code conventions](../integrations/powershell-code) (advanced users only).
+    * If your integration generates Incidents, follow the [Fetch Incidents](../fetching-incidents) guidelines.
+    * Make sure your commands make proper use of the [Context](../integrations/context-and-outputs), including [Context Standards](../integrations/context-standards-about) and [DBotScore](../integrations/dbot).
+    * Run and verify that the various linters we support pass as detailed [here](../integrations/linting).
+    * Make sure to create unit tests as documented [here](../integrations/unit-testing)
+    * Document your integration and automation as detailed [here](../documentation/readme_file) and [here](../documentation/documentation_tips).
+1. Make sure your Content Pack is properly [documented](../documentation/pack-docs) and read the [documentation best practices](../documentation/documentation_tips).
+1. Validate your content: the validation hook should run automatically every time you `git commit`. You can also run the validation manually by using [demisto-sdk validate](https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/validate/README.md). 
+1. As you build newer versions of your Content Pack, document your changes in a relevant release notes file as detailed [here](../documentation/release-notes).
+
+At this point you should be ready to submit a Pull Request! Check out again our [Contributing Checklist](../contributing/checklist), and for more details on the review process, refer to our [PR Conventions](../contributing/conventions) document.
```

Comment:
```
...and for more details on the review process, refer to our [PR Conventions](../contributing/conventions) document.
```
Let's put this in the `Before opening a Pull Request` section.

Reply:
We already have it in the "After you open a PR" so removed it from here.